assets/scss/components/_textarea.scss

```diff
@@ -0,0 +1,3 @@
+textarea {
+	min-height: 0;
```

Comment:
Having a `min-height` messes up the autoresizing functionality. I submitted a [PR in swarm-sasstools](https://github.com/meetup/swarm-sasstools/pull/7), so we can exclude this file from the PR altogether